demo_nodes_cpp/src/parameters/ros2param.cpp

```diff
@@ -139,6 +139,12 @@ int main(int argc, char ** argv)
       fprintf(stderr, "Failed to set parameter\n");
       return 1;
     }
+    auto results = set_parameters_result.get();
+    for (auto result : results) {
+      if (result.successful != true) {
```

Comment:
i don't see any other line using `!= true`. Can you please identify the lines you are referring to with links.